src/System.Transactions.Local/src/System/Transactions/DurableEnlistmentState.cs

```diff
@@ -11,128 +11,29 @@ namespace System.Transactions
     internal abstract class DurableEnlistmentState : EnlistmentState
     {
         // Double-checked locking pattern requires volatile for read/write synchronization
```

Comment:
>volatile [](start = 51, length = 8)

I am not clear on why the volatile keyword can be removed here. How does LazyInitializer.EnsureInitialized avoid requiring it to be volatile?

If it is, indeed, safe to remove volatile, then this comment should be removed.